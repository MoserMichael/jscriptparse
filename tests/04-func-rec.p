def fact(n) {
   if n<=1 {
      return 1
   }
   return n * fact(n-1)
}
println(fact(1))
println(fact(2))
println(fact(7))
