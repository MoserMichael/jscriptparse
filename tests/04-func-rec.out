1
2
5040
